packages/curve-ui-kit/src/shared/ui/ActionInfo.tsx

```diff
@@ -100,45 +100,48 @@ const ActionInfo = ({
         {label}
       </Typography>
 
-      {prevValue && (
-        <Stack direction="row" alignItems="center">
+      <Stack direction="row" alignItems="center" gap={Spacing.xs}>
+        {prevValue && (
           <Typography variant={prevValueSize[size]} color={prevValueColor ?? 'textTertiary'}>
             {prevValue}
           </Typography>
+        )}
 
+        {prevValue && (
           <ArrowForwardIcon
             sx={{
               width: IconSize.sm,
               height: IconSize.sm,
               color: (t) => t.palette.text.tertiary,
             }}
           />
-        </Stack>
-      )}
+        )}
 
-      <Stack direction="row" alignItems="center" gap={Spacing.xs}>
-        {valueLeft}
-
-        <WithSkeleton loading={!!loading}>
-          <Tooltip
-            title={valueTooltip}
-            placement="top"
-            slotProps={{
-              popper: {
-                sx: {
-                  userSelect: 'none',
-                  pointerEvents: 'none',
+        {/** Additional stack to add some space between left (icon), value and right (icon) */}
+        <Stack direction="row" alignItems="center" gap={Spacing.xxs}>
+          {valueLeft}
+
+          <WithSkeleton loading={!!loading}>
+            <Tooltip
+              title={valueTooltip}
+              placement="top"
+              slotProps={{
+                popper: {
+                  sx: {
+                    userSelect: 'none',
+                    pointerEvents: 'none',
+                  },
                 },
-              },
-            }}
-          >
-            <Typography variant={valueSize[size]} color={valueColor ?? 'textPrimary'}>
-              {!loading ? value : typeof loading === 'string' ? loading : MOCK_SKELETON}
-            </Typography>
-          </Tooltip>
-        </WithSkeleton>
-
-        {valueRight}
+              }}
+            >
+              <Typography variant={valueSize[size]} color={valueColor ?? 'textPrimary'}>
+                {!loading ? value : typeof loading === 'string' ? loading : MOCK_SKELETON}
```

Comment:
Should the MOCK_SKELETON be '00' since '0' is larger than '1'?

Reply:
We're not using monospace fonts so no digits has the same width. 10 might be too narrow, but 00 might be too wide depending on the real value, which we don't know. Probably a matter of pixels anyway.